bin/cli.js

```diff
@@ -156,11 +157,30 @@ import i18nTransform from '../dist/transform.js'
           if (!program.opts().silent) {
             console.log('  [warning] '.yellow + message)
           }
+          if (message.startsWith(KEY_REUSE_WARNING)) {
+            const key = message.substr(KEY_REUSE_WARNING.length)
+            const previousValue = warningMap[key] ?? 1
+            warningMap[key] = previousValue + 1
+          }
         })
         .on('finish', function () {
           if (!program.opts().silent) {
             console.log()
             console.log('  Stats:  '.cyan + count + ' files were parsed')
+            if (config.warnOnDuplicates && Object.keys(warningMap).length > 0) {
+              var keys = []
```

Comment:
I know there are inconsistencies in this file but I'd prefer we stick with `let/const`. Any reason you went for `var`?

Reply:
I went with `var` to match the `count` variable above it, but I can just switch both to `let` to fall more inline with the latest JS practices 👍🏼